documents/final-submission-document/final-submission-document.tex

```diff
@@ -102,19 +102,24 @@
 \vspace*{1ex}
 
 %Goal
-The goal of this thesis is to engineer a specialised and platform independent XMPP-Grid Broker that configures existing XMPP Controllers (XMPP server). The XMPP-Grid Broker should enable administrators to configure XMPP-Grids in a usable and productive way.
-
-The focus of the application should be set on portability, extensibility and the security aspects in a production environment.\\
+The goal of this thesis is to design and implement an XMPP-Grid Broker to configure existing Controllers (XMPP server), focussing on portability, extensibility and the security aspects in a production environment.
+The  Broker application should enable administrators to configure XMPP-Grids in a usable and productive way.\\
 \vspace*{1ex}
 
 %Result
-We propose the architecture of an XMPP-Grid Broker to configure XMPP Controllers and present an implementation based on the Angular5 framework.
+Our proposed architecture earmarks a purely client-side application that communicates typically through a regular reverse proxy with the XMPP Controller over WebSockets or HTTP streams (BOSH).
+Authentication is performed using mutual TLS authentication (client certificates) to conform to the IETF standard draft.
```

Comment:
I would remove client certificates here, as "mutual" also means verifying the server certificate (not only client certs).

Reply:
Yeah, sure, I only added it as a "denkstütze" to improve the readability